pkg/ddl/executor.go

```diff
@@ -643,19 +644,22 @@ func (e *executor) AlterTablePlacement(ctx sessionctx.Context, ident ast.Ident,
 	}
 
 	job := &model.Job{
+		Version:             model.JobVersion1,
```

Comment:
use VersionInUse(), it's not a multi schema type

Reply:
https://github.com/pingcap/tidb/blob/fd38012d745736e5e803ede9a780e4a5823c3de2/pkg/ddl/executor.go#L1994
This action is in subjob and need retain V1 currently.

Reply:
see fillMultiSchemaInfo, that part is useless